### PR TITLE
newfile: Create template for submission

### PR DESCRIPTION
**Pull request by kappapiana**
Created the template for submission to be filled in as a pull request comment

### CODE REVIEW

File: .github/.PULL_REQEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+## General information
+
+Use this template to provide (short) bio here and please add relevant
+information down below. To be pasted in the comment to your pull request.
+
+## self assessment:
```
Comment (purpleidea) [inline .github/.PULL_REQEST_TEMPLATE.md]: Let's just have this be the top level category (`Self Assessment:`) and include a small introductory message under it like: "Please fill out the following self assessment."

File: .github/.PULL_REQEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+- [ ] (1 pt/year, 3 pts max) active participation in a well-known legal
+discussion group
+> Your information goes here (and in the same place below)
```
Comment (purpleidea) [inline .github/.PULL_REQEST_TEMPLATE.md]: What's this line for?

File: .github/.PULL_REQEST_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
+## General information
```
Comment (purpleidea) [inline .github/.PULL_REQEST_TEMPLATE.md]: We can nix this section, see comment below.

**Review by purpleidea — CHANGES_REQUESTED** [review]
Great idea once again. File isn't named correctly though. Thanks!

**Comment by kappapiana** [comment]
I think all comments have been addressed.

File: .github/.PULL_REQUEST_TEMPLATE.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
+## self assessment:
```
Comment (purpleidea) [inline .github/.PULL_REQUEST_TEMPLATE.md]: capitalize please

File: .github/.PULL_REQUEST_TEMPLATE.md [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+
+Please fill out the following self assessment and provide context: .
```
Comment (purpleidea) [inline .github/.PULL_REQUEST_TEMPLATE.md]: why is there a trailing space, period??

File: .github/.PULL_REQUEST_TEMPLATE.md [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+- [ ] (1 pt) hold a law degree from a recognized university
+
+- [ ] (2 pt/publication, 2 pts max) publish an intermediate to advanced article
+about legal issues relevant to this list
+
+- [ ] (1 pt/referral, 2 pts max) receive a recommendation from someone on this
+list
+
+- [ ] (1 pt) demonstrate basic proficiency using the `git` tool
+
```
Comment (purpleidea) [inline .github/.PULL_REQUEST_TEMPLATE.md]: I don't think we need all the empty newlines between each list item.

**Review by purpleidea — CHANGES_REQUESTED** [review]
Looks pretty close, thanks!
A good practice can be to do a `git show` or `git diff ...` after your patch is ready, but before you push to look it over and see if it's exactly what you expect. Becoming detail oriented (when dealing with annoying programmers) is an invaluable skill! You're well on your way! Thanks for the patch.

**Comment by purpleidea** [comment]
Also, look at the example here: https://github.com/purpleidea/mgmt/tree/master/.github
Note the difference in how you named your file... (It's currently wrong)

File: .github/.PULL_REQUEST_TEMPLATE.md (hunk 5 above)
Comment (kappapiana) [reply .github/.PULL_REQUEST_TEMPLATE.md]: No F-- idea. corrected

**Comment by purpleidea** [comment]
LGTM, merged, thanks!

I would have merged sooner, but I didn't realize you had updated the branch. Protip: when you push or force-push, github doesn't notify you, so it's common to also write a little message when ready, eg: "okay, fixed, please review" or something similar. Thanks!

**Pull request merged**